app/views/users/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container" style="margin-bottom: 75px;">
+  <div class="d-flex align-items-center justify-content-left">
+    <%= link_to '◀︎', user_path(current_user), class: "btn btn-outline-primary rounded-pill d-block plastroc-outline-button mr-5" %>
+    <h2 class="mb-0">All users</h2>
+  </div>
+  <ul class="list-group">
+    <% @users.order(:xp).reverse.each do |user| %>
```

Comment:
As we saw this morning, I think this operation should be done in the controler to retreive a @sorted_users directly in the view